releasenotes/notes/horizon-extensions-in-tree-91f2ed817fde11aa.yaml

```diff
@@ -2,3 +2,8 @@
 other:
   - The horizon extensions are now provided in the
     rpc-openstack repository.
+upgrade:
```

Comment:
@sigmavirus24 makes a good point. As I understand it, the release note is listed under the first tag against which it appears. I think the original release note appears in r13.0.0rc1 and so I believe that means we should have a separate note. If the change was to correct or clarify the existing comment then it would makes sense but in this case I believe it should be separate.